README.md

```diff
@@ -27,9 +45,71 @@ While setting up cassandra instance, make sure the data directories of cassandra
 Testing
 -------
 
-We rely on docker containers for integration tests.
+The test framework is set up to run 4.1 and 5.0 (Trunk) tests (see `TestVersionSupplier.java`) by default.  
+You can change this via the Java property `cassandra.sidecar.versions_to_test` by supplying a comma-delimited string.
+For example, `-Dcassandra.sidecar.versions_to_test=4.0,4.1,5.0`.
 
-The only requirement is to install and run [Docker](https://www.docker.com/products/docker-desktop/) on your test machine.
+In order for tests to run successfully under JDK11, you'll need to add the following JVM arguments to your test runner of choice.
+You should also set your test framework to fork a new process at least every class, if not every method, as there are still
+a few unresolved memory-related issues in the in-jvm dtest framework.
+```
+-Djdk.attach.allowAttachSelf=true
+-XX:+UseConcMarkSweepGC
+-XX:+CMSParallelRemarkEnabled
+-XX:SurvivorRatio=8
+-XX:MaxTenuringThreshold=1
+-XX:CMSInitiatingOccupancyFraction=75
+-XX:+UseCMSInitiatingOccupancyOnly
+-XX:CMSWaitDuration=10000
+-XX:+CMSParallelInitialMarkEnabled
+-XX:+CMSEdenChunksRecordAlways
+--add-exports
+java.base/jdk.internal.misc=ALL-UNNAMED
+--add-exports
+java.base/jdk.internal.ref=ALL-UNNAMED
+--add-exports
+java.base/sun.nio.ch=ALL-UNNAMED
+--add-exports
+java.management.rmi/com.sun.jmx.remote.internal.rmi=ALL-UNNAMED
+--add-exports
+java.rmi/sun.rmi.registry=ALL-UNNAMED
+--add-exports
+java.rmi/sun.rmi.server=ALL-UNNAMED
+--add-exports
+java.rmi/sun.rmi.transport.tcp=ALL-UNNAMED
+--add-exports
+java.sql/java.sql=ALL-UNNAMED
+--add-opens
+java.base/java.lang.module=ALL-UNNAMED
+--add-opens
+java.base/java.net=ALL-UNNAMED
+--add-opens
+java.base/jdk.internal.loader=ALL-UNNAMED
+--add-opens
+java.base/jdk.internal.ref=ALL-UNNAMED
+--add-opens
+java.base/jdk.internal.reflect=ALL-UNNAMED
+--add-opens
+java.base/jdk.internal.math=ALL-UNNAMED
+--add-opens
+java.base/jdk.internal.module=ALL-UNNAMED
+--add-opens
+java.base/jdk.internal.util.jar=ALL-UNNAMED
+--add-opens
+jdk.management/com.sun.management.internal=ALL-UNNAMED
+-Dcassandra-foreground=yes
+-Dcassandra.config=file:///Users/drohrer/p/apache-cassandra-bare/cassandra-4.1/conf/cassandra.yaml
+-Dcassandra.jmx.local.port=7199
+-Dcassandra.logdir=/Users/drohrer/p/apache-cassandra-bare/cassandra-4.1/data/logs
+-Dcassandra.reads.thresholds.coordinator.defensive_checks_enabled=true
+-Dcassandra.storagedir=/Users/drohrer/p/apache-cassandra-bare/cassandra-4.1/data
+-Dcassandra.triggers_dir=/Users/drohrer/p/apache-cassandra-bare/cassandra-4.1/conf/triggers
+-Djava.library.path=/Users/drohrer/p/apache-cassandra-bare/cassandra-4.1/lib/sigar-bin
+-Dlogback.configurationFile=file:///Users/drohrer/p/apache-cassandra-bare/cassandra-4.1/conf/logback.xml
```

Comment:
Normalize the paths?

Reply:
They actually aren't necessary (the paths - see below for why I include this in the readme). I'll remove the extra parameters.